src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -45,5 +47,20 @@ public List<IexLastTradedPrice> getLastTradedPriceForSymbols(final List<String>
     }
   }
 
+  /**
+   * Get historical pricing data for the given symbol on the given date
+   *
+   * @param symbol the symbol to retrieve data about
+   * @param date the date which should be queried
+   * @return historical pricing data for the given symbol on the given date
+   */
+  public List<IexHistoricalPrice> getHistoricalPrice(final List<String> symbol, final List<String> date) {
+    if ((symbol.size() == 1) && (date.size() == 1)) {
```

Comment:
Since we only expect one symbol and one date, it would be better to take in a single value for each rather than a List.